src/mixins/InitHandler.js

```diff
@@ -0,0 +1,27 @@
+import LoadingIndicator from '../components/admin/qrcode/LoadingIndicator'
```

Comment:
InitHandler.js is specialised for QR code ? if so then it should be QRCodeInitHandler.js